webapp/channels/.eslintrc.json

```diff
@@ -37,19 +37,55 @@
     "eol-last": ["error", "always"],
     "import/no-unresolved": 2,
     "import/order": [
-      0,
+      2,
       {
-        "newlines-between": "always-and-inside-groups",
+        "newlines-between": "always",
         "groups": [
           "builtin",
           "external",
-          [
-            "internal",
-            "parent"
-          ],
+          "internal",
           "sibling",
+          "parent",
           "index"
-        ]
+        ],
+        "pathGroups": [
+          {
+            "pattern": "react*{,/**}",
+            "group": "builtin",
+            "position": "before"
+          },
```

Comment:
0/5 on treating React dependencies differently from other external dependencies

Reply:
I got the idea while I was looking to fix one bug I had around. My point with this change is that in general, in React files, the "import React from 'react'" tends to be up top. I like the idea and how the components look like, but 0/5. Happy to remove this rule.

Reply:
In that case, I'd say we remove this rule for simplicity's sake. We only have a handful of non-MM, non-React-related external dependencies that are widely used, so I don't think we need to separate them out